test/Project.toml

```diff
@@ -8,6 +8,7 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
 Aqua = "0.8.9"
+GradedUnitRanges = "0.1.1"
```

Comment:
```suggestion
```
I don't think we need this here as well, it is implicit since it is set in the package's `Project.toml`. We only need to add compat entries here for packages that are only used in the tests and not in the main package.